testing/trino-faulttolerant-tests/src/test/java/io/trino/faulttolerant/BaseFailureRecoveryTest.java

```diff
@@ -251,7 +251,11 @@ protected void testSelect(String query, Optional<Session> session, Consumer<Quer
     @Test(invocationCount = INVOCATION_COUNT)
     public void testUserFailure()
     {
-        assertThatThrownBy(() -> getQueryRunner().execute("SELECT * FROM nation WHERE regionKey / nationKey - 1 = 0"))
+        Session withoutPushdown = Session.builder(this.getSession())
+                .setSystemProperty("allow_pushdown_into_connectors", "false")
+                .build();
```

Comment:
should we add a comment why we need this or is this relatively obvious? I'll defer to @losipiuk / @mwd410.

Reply:
I added comments for both casses